src/microbe_stage/CompoundCloudPlane.cs

```diff
@@ -55,6 +59,8 @@ public class CompoundCloudPlane : CSGMesh, ISaveLoadedTracked
     // Called when the node enters the scene tree for the first time.
     public override void _Ready()
     {
+        mucilage = SimulationParameters.Instance.GetCompound("mucilage");
```

Comment:
I think CompoundCloudPlane shouldn't know about a specific compound. Perhaps I think a better a way could be to add a new `absorbable` property (if there isn't already) to the compounds which is set false for the mucilage. This property will then be checked higher up in the microbe's own compound absorbing handling.